patches/minecraft/net/minecraft/client/entity/EntityPlayerSP.java.patch

```diff
@@ -111,3 +111,12 @@
          boolean flag4 = (float)this.func_71024_bL().func_75116_a() > 6.0F || this.field_71075_bZ.field_75101_c;
  
          if (this.field_70122_E && !flag1 && !flag2 && this.field_71158_b.field_192832_b >= 0.8F && !this.func_70051_ag() && flag4 && !this.func_184587_cr() && !this.func_70644_a(MobEffects.field_76440_q))
+@@ -916,7 +941,7 @@
+         {
+             ItemStack itemstack = this.func_184582_a(EntityEquipmentSlot.CHEST);
+ 
+-            if (itemstack.func_77973_b() == Items.field_185160_cR && ItemElytra.func_185069_d(itemstack))
++            if ((itemstack.func_77973_b() == Items.field_185160_cR && ItemElytra.func_185069_d(itemstack)) || itemstack.func_77973_b() instanceof net.minecraftforge.items.IElytra)
```

Comment:
IElytras should have ways to determine if the item stack can be used for flying (Like vanilla elytras can do that)